src/main/java/teammates/common/datatransfer/UserInfoCookie.java

```diff
@@ -14,9 +17,12 @@
     private String userId;
     private String verificationCode;
 
+    private Instant expiryTime;
```

Comment:
Although this should work, for serialized data, a more primitive type works better. `long` is the obvious choice whereby the epoch millis time is stored instead.

Reply:
Noted!